### PR TITLE
Add Services to Check Results Page

### PR DESCRIPTION
**Pull request by ejsundstr**
Baby's First Cabot PR

Still probably needs some work, including testing this but I think this is most of it.

Going to use this to learn how to do all the caboty dancin'

image

### CODE REVIEW

File: cabot/cabotapp/views.py [hunk 1]
```diff
@@ -481,6 +481,9 @@ def render_to_response(self, context, *args, **kwargs):
             context = {}
         context['checkresults'] = self.object.statuscheckresult_set.order_by(
             '-time_complete')[:100]
+        statuscheck_id = self.objects.statuscheckresult_set.first().status_check.id
```
Comment (Aloshi) [inline cabot/cabotapp/views.py]:
```suggestion
        statuscheck_id = self.object.id
```

File: cabot/templates/cabotapp/statuscheck_detail.html [hunk 2]
```diff
@@ -26,6 +26,14 @@
 <hr>
 <div class="row">
   <div class="col-xs-12">
+    <div class="col-xs-11"><h3>Associated Services</h3></div>
+    {% for service in services %}
```
Comment (Aloshi) [inline cabot/templates/cabotapp/statuscheck_detail.html]: Missing `{% endfor %}`?

File: cabot/cabotapp/views.py (hunk 1 above)
Comment (ejsundstr) [reply cabot/cabotapp/views.py]: Ahh I have direct access to the check model in that view.  Cool!

File: cabot/templates/cabotapp/statuscheck_detail.html (hunk 2 above)
Comment (ejsundstr) [reply cabot/templates/cabotapp/statuscheck_detail.html]: totally!

**Pull request merged**